backend/app/core/config.py

```diff
@@ -96,18 +96,19 @@ def _set_default_emails_from(self) -> Self:
     def emails_enabled(self) -> bool:
         return bool(self.SMTP_HOST and self.EMAILS_FROM_EMAIL)
 
-    EMAIL_TEST_USER: EmailStr = "test@example.com"
+    EMAIL_TEST_USER: EmailStr
```

Comment:
the idea is also to check if we are using these variables and if not then remove it